feature/office/src/main/java/com/iteneum/office/presentation/ui/Office.kt

```diff
@@ -18,54 +20,72 @@ import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.text.style.TextIndent
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.viewmodel.compose.viewModel
 import com.iteneum.designsystem.components.LpOutlinedButton
 import com.iteneum.designsystem.theme.LeasePertTheme
 import com.iteneum.office.R
+import com.iteneum.office.presentation.viewmodel.OfficeViewModel
 
 /**
  * This function creates the Office screen UI
  *With the Outlined buttons and the company information
+ * @param vm injects the viewModel into the UI
  * @author Yaritza Moreno
  * Comment
  */
 @Composable
-fun OfficeUI() {
-
+fun OfficeUI(vm: OfficeViewModel = hiltViewModel())
+{
     Column() {
 
         //Get sizes from LeasePertTheme archive
         val sizes = LeasePertTheme.sizes
 
-        Text(text = stringResource(id = R.string.LPInfo),
+        val context: Context
+
+        Text(
+            text = stringResource(id = R.string.LPInfo),
             modifier = Modifier
                 .width(sizes.extraSize124)
                 .height(sizes.regularSize),
-            style = TextStyle(textAlign = TextAlign.Justify, color = Color.Gray,
+            style = TextStyle(
+                textAlign = TextAlign.Justify, color = Color.Gray,
```

Comment:
Would you add the property color to the next line to have a good structure in the code, please?